corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -357,7 +357,7 @@ def __unicode__(self):
         return "%s synced on %s (%s)" % (self.user_id, self.date.date(), self.get_id)
 
 
-class OwnershipCleanliness(models.Model):
+class OwnershipCleanlinessFlag(models.Model):
```

Comment:
So after you run the migration we'll still just be updating any flags already in the DB and not adding new ones correct?